internal/cmd/destroy/cmd.go

```diff
@@ -0,0 +1,22 @@
+package destroy
+
+import (
+	"github.com/spf13/cobra"
+)
+
+// DestroyCmd destroys generated code
+var cmd = &cobra.Command{
```

Comment:
same here, just move this declaration to the `Cmd()` function instead of having a global variable.